packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -334,7 +336,10 @@ function matchObjects(
 	expected: ITelemetryPropertiesExt,
 ): boolean {
 	for (const [expectedKey, expectedValue] of Object.entries(expected)) {
-		const actualValue = actual[expectedKey];
+		const actualValue:
+			| TelemetryEventPropertyTypeExt
+			| Tagged<TelemetryEventPropertyTypeExt>
+			| undefined = actual[expectedKey];
```

Comment:
Woah is this really what the formatter wants?  I am not a fan of how the RHS of the `=` is on the same line as the end of that multi-line type.  Weird, oh well.